docs/pages/workflow/development-mode.mdx

```diff
@@ -6,19 +6,21 @@ sidebar_title: Development and production
+Your project will always run in two modes: **development** or **production**. By default, running your project locally with `npx expo start` runs it in development, whereas a published project (with `eas update`), or any standalone app, will run in production.
 
-Production mode [minifies your code](https://www.imperva.com/learn/performance/minification/) and better represents the performance your app will have on end users' devices. Development mode includes useful warnings and gives you access to tools that make development and debugging easier. Let's look at each of these modes more in detail and learn how you can toggle between them.
+**Development mode** includes useful warnings and gives you access to tools that make development and debugging easier. **Production mode** [minifies your code](https://www.imperva.com/learn/performance/minification/) and better represents the performance your app will have on end users' devices. Let's look at each of these modes more in detail and learn how you can toggle between them.
 
-## Development Mode
+## Development mode
 
-React Native includes some very useful tools for development: remote JavaScript debugging in Chrome, live reload, hot reloading, and an element inspector similar to the beloved inspector that you use in Chrome. If you want to see how to use those tools, see our [debugging documentation](debugging.mdx). Development mode also performs validations while your app is running to give you warnings if, for example, you're using a deprecated property or if you forgot to pass a required property into a component. This video shows the Element Inspector and Performance Monitor in action, on both Android and iOS simulators:
+React Native includes some very useful tools for development: remote JavaScript debugging in Chrome, live reload, hot reloading, and an element inspector similar to the beloved inspector you use in Chrome. If you want to see how to use those tools, see [Debugging](/workflow/debugging).
+
+Development mode also performs validations while your app is running to give you warnings if, for example, you're using a deprecated property or if you forgot to pass a required property into a component. The video below shows the Element Inspector and Performance Monitor in action on both Android and iOS simulators:
 
 <Video file="dev-prod/devMode.mp4" />
 
 > **This comes at a cost: your app runs slower in development mode.**
 >
-> You can toggle it on and off with the Expo CLI, see [production mode](#production-mode). When you switch it, just close and re-open your app for the change to take effect. **Any time you are testing the performance of your app, be sure to disable development mode**.
+> You can toggle it on and off with the Expo CLI, see [Production mode](#production-mode). When you switch it, just close and re-open your app for the change to take effect. **Any time you are testing your app's performance, be sure to disable development mode**.
```

Comment:
'toggle' _or_ 'switch'

Reply:
I'll rephrase this part. Found another issue.